doc/source/conf.py

```diff
@@ -69,11 +70,13 @@
 # Favicon
 html_favicon = ansys_favicon
 
-extensions = [
-    "sphinx_design",
-    "sphinx_copybutton",
-    "sphinxcontrib.mermaid",
-]
+extensions = ["sphinx_design", "sphinx_copybutton", "sphinxcontrib.mermaid", "sphinx_jinja"]
```

Comment:
Can we have the extensions in a multiline implementation? Makes it more readable
```suggestion
extensions = [
    "sphinx_design",
    "sphinx_copybutton",
    "sphinxcontrib.mermaid", 
    "sphinx_jinja",
]
```